### ISSUE TITLE
inclined elevators

### ISSUE THREAD

**Issue by troposphhere** [gravitystorm/openstreetmap-carto#4261]
Hi,
This is a follow-up to #1953 . It was closed at the time, deciding that inclined elevators are1) rare and 2) could be seen as a type of funicular. After a discussion on the tagging mailing list, I think there are good reasons to consider supporting them as separate elements, represented by a way tagged with `highway=elevator`.

First of all, this usage is supported by the [wiki:](https://wiki.openstreetmap.org/wiki/Tag:highway%3Delevator#How_to_Map_as_a_Way). Those elevators are actually very common in Sweden, and maybe in other places. There are currently [3200 instances](https://taginfo.openstreetmap.org/tags/highway=elevator) of those in the database.

An example can be seen on the left of the main picture in [this article](http://nyttiflempan.sh.se/flemingsberg/2015-04-21/S%C3%B6dert%C3%B6rns-rulltrappa-en-huvudv%C3%A4rk-f%C3%B6r-kommunen-12389.html). As you can see, this is really just a standard elevator that moves on a inclined path. It is very different from a funicular.

### Expected behavior
Inclined elevators are displayed (perhaps the same as escalators in a different colour?)

### Actual behavior
Inclined elevators are not displayed at all.

### Links and screenshots illustrating the problem
Two examples are in [Flemingsberg](https://www.openstreetmap.org/way/880291343) and [Hamburg](https://www.openstreetmap.org/way/814305826).

The attached picture illustrates the current situation: There are three escalators and an inclined elevator in parallel. The elevator, in the south-west of the escalators, is invisible.
![elevator](https://user-images.githubusercontent.com/17337046/101171776-8c06e100-3640-11eb-9ff0-0599d2fb2fe0.png)

**Comment by imagico** [comment]
Duplicate of #1953.

**Comment by jeisenbe** [comment]
@troposphhere thank you for mentioning this issue again. Most likely you were not aware of the previous issue about this problem, since it was closed. When you search for old issues, it can be helpful to remove the default "is:open” field from the search box.

Please add your thoughts on issue #1953, which is now open again.

**Comment by troposphhere** [comment]
Sorry, I was aware of it but assumed it was better to start a new issue since the other was closed. I will post there!
